llvm/lib/Target/PowerPC/PPCInstrFuture.td

```diff
@@ -82,3 +82,23 @@ let Predicates = [HasVSX, IsISAFuture] in {
                                    "stxvprll $XTp, $RA, $RB", IIC_LdStLFD, []>;
   }
 }
+
+// Load/Store VSX Vector with Right Length Left-justified.
+foreach Ty = [v4i32, v2i64] in {
+  def : Pat<(Ty (int_ppc_vsx_lxvrl addr:$RA, i64:$RB)),
+            (LXVRL memr:$RA, g8rc:$RB)>;
```

Comment:
I might be missing something in my understanding, but is the `memr` and `g8rc` necessary just for this pattern?